pytest/tests/sanity/upgradable.py

```diff
@@ -164,8 +164,13 @@ def test_upgrade() -> None:
     assert 'error' not in res, res
     assert 'Failure' not in res['result']['status'], res
 
-    hex_account_balance = int(
-        nodes[0].get_account(hex_account_id)['result']['amount'])
+    # We have seen test failures, where account was not found.
+
+    response = nodes[0].get_account(hex_account_id)
+
+    assert 'result' in response, f'got error from node: {response["error"]}'
```

Comment:
By the way, looking at the rest of the code `assert 'error' not in res, res` is actually more consistent.